client/src/app/ask-feedback-form/ask-feedback-form.component.css

```
@@ -1,6 +1,46 @@
+.container {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+}
+.ask-logo {
+    display: flex;
+    justify-content: center;
+}
+.header-section {
+    display: flex;
+    flex-direction: column;
+    margin-left: 450px;
+}
+.title{
+    display: flex;
+ flex-direction: row;
```

```suggestion
    display: flex;
     flex-direction: row;
```
Toujours des soucis de formatages.